src/lib/main.js

```diff
@@ -123,20 +123,17 @@ module.exports = async (argv, env) => {
     return -1;
   }
 
-  const projectName = fs.path.basename(fs.path.resolve('.'));
-
-  if (instanceUrl) {
-    const productionUrlMatch = instanceUrl.href.match(/^https:\/\/(?:[^@]*@)?(.*)\.(app|dev)\.medicmobile\.org(?:$|\/)/);
-    if (productionUrlMatch &&
-        productionUrlMatch[1] !== projectName &&
-        productionUrlMatch[1] !== 'alpha') {
-      warn(`Attempting to use project for \x1b[31m${projectName}\x1b[33m`,
-          `against non-matching instance: \x1b[31m${redactBasicAuth(instanceUrl.href)}\x1b[33m`);
-      if(!readline.keyInYN()) {
-        error('User failed to confirm action.');
-        return -1;
+  const currentFolderName = fs.path.basename(fs.path.resolve('.'));
+  if (instanceUrl && instanceUrl.hostname.endsWith('.medicmobile.org')) {
+      const [urlProjectName] = instanceUrl.hostname.split('.', 1);
+      if (urlProjectName !== currentFolderName && urlProjectName !== 'alpha') {
```

Comment:
I think `alpha` has been decommission since the transition to 3.0 arch. Most testing is done on `gamma`. I think we can swap them here to allow gamma to receive random configs without warning?